svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/(file-view)/[...tree]/+page.svelte

```diff
@@ -3,17 +3,18 @@
   import FileExplorerContainer from '$lib/components/FileExplorer/FileExplorerContainer/FileExplorerContainer.svelte';
   import FileExplorerNav from '$lib/components/FileExplorer/FileExplorerNav/FileExplorerNav.svelte';
   import FileExplorerReadme from '$lib/components/FileExplorer/FileExplorerReadme/FileExplorerReadme.svelte';
-  import type { LayoutServerData } from './$types';
+  import type { PageServerData } from './$types';
 
-  export let data: LayoutServerData;
-  const { repoInfo, username, repo } = data;
-  const { description, website, tags, readme, tree, activeBranch } = repoInfo;
+  export let data: PageServerData;
+  console.log('DATA', data);
+  const { repoInfo, username, repo, readme, tree } = data;
```

Comment:
This is a bit of a "svelteism" but if you destructure an object (or array) and you need the properties to be reactive you need to use the "reactive destructuring" syntax [described in the svelte docs](https://svelte.dev/docs#component-format-script-3-$-marks-a-statement-as-reactive), which is basically, _in a reactive statement destructure as an expression_.

```suggestion
  $: ({ repoInfo, username, repo, readme, tree } = data);
```

...once you change them to this you should see the components updating correctly. It's a tricky little thing but now that you know it it'll only bite you one or two more times.

We probably want to look for other instances of destructuring in svelte components and change them to this style too.